src/test/java/teammates/test/pageobjects/AdminHomePage.java

```diff
@@ -95,4 +96,9 @@ public String getMessageFromResultTable(int index) {
         return getCellValueFromDataTable(index, 5);
     }
 
+    public String getJoinLink(String messageText) {
+        WebElement link = browser.driver.findElement(By.linkText("[Join Link]"));
```

Comment:
Is it possible to extract this string `"[Join Link]"` into a constant for use both here and in the code above? Otherwise, this test may be quite brittle: if the text for the code in `AdminInstructorAccountAddAction` changes in the future, this will result in failing test cases.